src/bullAndCowCounter.js

```diff
@@ -0,0 +1,24 @@
+'use strict';
+
+function bullAndCowCounter(arrayOfRandomNumbers, arrayOfUserNumbers) {
+  let bull = 0;
+  let cow = 0;
+
+  for (let i = 0; i < arrayOfUserNumbers.length; i++) {
+    if (arrayOfRandomNumbers.includes(arrayOfUserNumbers[i])) {
+      cow++;
+    }
+
+    if (arrayOfRandomNumbers.indexOf(arrayOfUserNumbers[i]) === i) {
+      cow--;
+      bull++;
```

Comment:
a bit of strange approach, instead you can check if array includes, assign to a variable, and then if includes and `i === i` bull++, else cow++